src/components/TodoItem/TodoItem.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import { deleteTodo, editTodo, toggleTodo } from "../../redux/actions";
+import styles from "./TodoItem.module.css";
+import Input from "../Input/Input";
+import { BsCheckSquare, BsPen, BsTrash, BsXSquare } from "react-icons/bs";
+
+export default function TodoItem({ todo }) {
+  const dispatch = useDispatch();
+
+  const [edit, setEdit] = useState(false);
+  const [title, setTitle] = useState(todo.title);
+
+  const toggleTodoTask = () => {
+    const action = toggleTodo(todo.id);
```

Comment:
you can use destructuring in this case : const { id } = todo;
I think it will works with styles.edit, styles.button, etc., too.

Reply:
fixed